.github/workflows/update-config-index.yml

```diff
@@ -0,0 +1,100 @@
+# This workflow checks if any config files were changed without updating the index
+# If so, it asks the author it if should be done automatically
+
+name: Update Devices Index
+
+on:
+  pull_request:
+    types: [opened, edited]
+  issue_comment:
+    types: [created]
+
+jobs:
+  check_changed:
+    runs-on: ubuntu-latest
+    # Use this job to check whether the next jobs need to run
+    outputs:
+      config_changed: ${{ steps.check_changed.outputs.CHANGED }}
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Fetch all branches
+        run: |
+          git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
+
+      - name: Check if any config files changed
+        id: check_changed
+        run: |
+          if $(git diff origin/master HEAD --name-only | grep -q packages/config/config/devices)
+          then
+            echo "Devices DB has been updated"
+            echo "::set-output name=CHANGED::true"
+          else
+            echo "No device changes detected"
+          fi
+
+  # ----
+
+  check_and_update_index:
+    runs-on: ubuntu-latest
+    needs: check_changed
+    # Skip this job if the config files were not touched
+    if: needs.check_changed.outputs.config_changed
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Install Node.js
+        if: steps.check_changed.outputs.CHANGED == 'true'
```

Comment:
Is this if still needed even if the jobs are splitted?

Reply:
Probably not - we didn't get that far during tests